charts/geoweb-opmet-backend/README.md

```diff
@@ -111,6 +115,16 @@ The following table lists the configurable parameters of the Opmet backend chart
 | `opmet.db_secretType` | Type to db secret | `secretsmanager` |
 | `opmet.db_secretPath` | Path to db secret | |
 | `opmet.db_secretKey` | Key of db secret | |
+| `opmet.ssh_secret` | Secret containing base64 encoded SSH private key | |
+| `opmet.ssh_secretName` | Name of db secret | `opmet-publisher-ssh-key` |
+| `opmet.ssh_secretType` | Type to db secret | `secretsmanager` |
+| `opmet.ssh_secretPath` | Path to db secret | |
+| `opmet.ssh_secretKey` | Key of db secret | |
+| `opmet.ssh_passphrase_secret` | Secret containing base64 encoded SSH private key passphrase | |
+| `opmet.ssh_passphrase_secretName` | Name of db secret | `opmet-publisher-ssh-passphrase` |
+| `opmet.ssh_passphrase_secretType` | Type to db secret | `secretsmanager` |
+| `opmet.ssh_passphrase_secretPath` | Path to db secret | |
+| `opmet.ssh_passphrase_secretKey` | Key of db secret | |
```

Comment:
Forgot to update descriptions here. Looks good otherwise 👍

Reply:
Updated descriptions